Makefile

```diff
@@ -28,9 +28,11 @@ install:
 	install -D -m 755 $(BINARY) $(BINDIR)/$(BINARY)
 	install -D -m 644 docker-lvm-plugin.8 ${MANINSTALLDIR}/man8/docker-lvm-plugin.8
 
+.PHONY: test
+test:
```

Comment:
As noted below, the tests are destructive. Since `make test` is a common development pattern, how about renaming this target to something obscure and adding a `make test` target that \@echo's a loud warning, then gives the real test target name?

Reply:
I would like to keep `make test` (like you mentioned it's a common development pattern) since I don't want to make it more difficult for the user to run tests.

Having said that, This is more of a **WARNING** and if a user runs these tests on his/her host machine, tests will clean up everything appropriately and leave the system in the same state from where it started.

Let me look into if I can add a `LOUD` warning before starting the tests.

Reply:
No worries - a warning in `make test` would, I think, make things worse because someone's first reaction is likely to be to hit ^C -- which would be worse than letting the tests finish and clean up.

Reply:
I agree. Keeping it as-is for now.